app/javascript/packs/application.js

```diff
@@ -87,16 +83,7 @@ $(function () {
 
   $('.data_table').DataTable({
     searching: true,
-    language: {
-      url: '//cdn.datatables.net/plug-ins/9dcbecd42ad/i18n/German.json'
-    },
     order: [[0, 'desc']],
-    columnDefs: [
-      {
-        bSortable: false,
-        targets: 'nosort'
-      }
-    ]
```

Comment:
this should have not been removed! this is not related to date/time/moment, but to be able to mark some columns as not sortable. with your change this results in

<img width="167" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-cms/assets/1942953/ecb43dd5-1561-4cc4-81a6-9ac9ebc6f56a">

where the button column would be sortable now.
i will revert that.

Reply:
ah... thank you for the explanation